app/api/templates/specs/generatedIdPropertyAutoFiller.spec.ts

```diff
@@ -0,0 +1,70 @@
+import entities from 'api/entities';
```

Comment:
Is there a reason for this test being in a different route than the module is testing?

Reply:
Yes, it's needed to check the indexation as part of the process. Probably in the future when we address the problem with indexation we can relocate this.